ruby/.rubocop.yml

```diff
@@ -5,6 +5,7 @@ AllCops:
     - "config/**/*"
     - "Guardfile"
     - "Rakefile"
+    - "node_modules/**/*"
```

Comment:
Hi @kelvin8773 !

Thank you for this contribution! 🙇 
Could you also add this dir here to ignored files in Stickler config: https://github.com/microverseinc/linters-config/blob/master/ruby/.stickler.yml#L5 ?

Reply:
ok, updated, please check!